perfkitbenchmarker/pkb.py

```diff
@@ -331,23 +332,17 @@ def _LogCommandLineFlags():
   logging.info('Flag values:\n%s', '\n'.join(result))
 
 
-def RunBenchmarks(publish=True):
-  """Runs all benchmarks in PerfKitBenchmarker.
-
-  Args:
-    publish: A boolean indicating whether results should be published.
+def SetUpPKB():
+  """Set globals and environment variables for PKB.
 
-  Returns:
-    Exit status for the process.
+  Having this function separate from functions that do any work lets
+  us call this to set up integration tests.
   """
-  if FLAGS.version:
-    print version.VERSION
-    return
 
   for executable in REQUIRED_EXECUTABLES:
     if not vm_util.ExecutableOnPath(executable):
       logging.error('Could not find required executable "%s".' % executable)
-      return 1
+      raise errors.Error()
```

Comment:
Maybe UserSetupError or some other more specific type. Also, add message to exception. Same with others below.